app/helpers/routes/getCommencementDate.js

```diff
@@ -17,5 +17,6 @@ export const getCommencementDate = async ({
   };
 
   const sessionData = { req, key: sessionKeys.plannedDeliveryDate };
-  return getFromSessionOrApi({ sessionData, sessionManager, apiCall });
+  const date = getFromSessionOrApi({ sessionData, sessionManager, apiCall });
+  return date.commencementDate ? date.commencementDate : date;
```

Comment:
When will `date.commencementDate` be `undefined`?

Reply:
if it gets it from session then it will only have the date as a string

Reply:
Ah, I see. Might it be worth tweaking `apiCall` to something like this instead? (Caveat: this code may not be 100% correct!)

```javascript
const apiCall = async () => {
    const orgId = req.user.primaryOrganisationId;
    const { commencementDate } = getCommencementDateFromApi({
      orgId,
      accessToken,
      logger,
    });

   return commencementDate;
};
```

What do you think?

Reply:
yeah bit easier than having date.commencementdate to read